rclcpp_components/cmake/rclcpp_components_add_node.cmake

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Create a target which builds a shared library and also creates a target
+# which builds an executable
```

Comment:
Since `ARGS_TARGET` and `ARG_SRCS` are not keyword args would document usage before describing the arguments, e.g.

```
usage: rclcpp_components_add_node(<target> [EXCLUDE_FROM_ALL] <source1> [source2 ...])